.github/contributing.md

```diff
@@ -0,0 +1,213 @@
+16. **Developer C** merges the release PR which activates a Github workflow that tags the main branch with 
+the next version and creates the release on Github 
+
+
+Let's solidify this with an example.
+
+## 👨🏻‍💻 Develop
+
+This assumes you've already setup your development environment. If not, 
+follow the steps in [Setup Dev Environment](#setup-dev-environment)
+
+### Create a branch
+```shell
+# Checkout your local main branch
+git checkout main
+
+# Update your local main branch to mirror the main branch on Github
+git pull
+
+# Create a feature branch to work on. Name it something short and relevant 
+# to the change
+git checkout -b add-patient-birth-sex 
+```
+
+### Commit the change 
+
+After you've made the change you want on your local machine its time 
+to commit that change to the project repo. 
+
+#### Emoji Commits
+This project uses emoji based commits where we prefix the commit message with 
+an emoji code that is relevant to that change. 
+
+This is actually somewhat of a standard practice in the industry, to the point 
+where standard emoji codes for various types of changes are published on the [gitmoji]() 
+website. 
+
+Here's an example commit message for adding something new:
+
+```shell
+# Stage the file(s) to commit
+$ git add <file path to the file you want to commit>
+
+# Commit the file(s) to the local repo
+$ git commit -m ":sparkles: Add new birthSex extension to Patient" 
+```
+
+### ⚠️  Important! - Rebase
+If another developer has merged their code (Pull Request) while you're working
+on your change, you will need to update your branch with those changes before
+you continue developing. To do that, you will need to [rebase your branch]()
+on the main branch like this:
+
+```shell
+# Checkout your local main branch
+git checkout main
+
+# Update your local main branch to mirror the main branch on Github
+git pull
+
+# Update your local branch with the changes on Github main branch 
+git checkout add-patient-birth-sex 
+git rebase main 
+
+# Push your updated branch to Github 
+git push -f
+```
+
+If you do this often you will rarely have conflicts.
+
+
+## 📝 Pull Request
+
+Once you're happy with your changes, its time to [open a Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request?tool=webui)
+so that others can review your code and upon their approval you can merge your branch
+into the main branch.
+
+### Title
+Similar to commit messages, the standard practice for this project is to 
+prefix the PR title with the appropriate emoji from Gitmoji. 
+
+For example:
+
+**✨ Add new birthSex field to Patient**
+
+### Description
+When you open a pull request you will notice that the description is populated
+with a "Motivation" and "Approach" section. Please be sure to fill these out 
+so that reviewers can quickly and easily understand the purpose of the PR and 
+the changes.
+
+## Squash and Merge
```

Comment:
The consensus in the NCPI code-a-thon is that we avoid squashing to preserve commit history.

Reply:
We decided that preserving nitty gritty commit history is not important and we believe that squash merge will make rebasing a lot easier